app/uk/gov/hmrc/tai/config/ApplicationConfig.scala

```diff
@@ -82,6 +82,7 @@ class MongoConfig @Inject()(val runModeConfiguration: Configuration) extends Bas
   lazy val mongoEncryptionEnabled: Boolean =
     runModeConfiguration.getOptional[Boolean]("mongo.encryption.enabled").getOrElse(true)
   lazy val mongoTTL: Int = runModeConfiguration.getOptional[Int]("tai.cache.expiryInSeconds").getOrElse(900)
+  lazy val mongoTTLUpdateIncome: Int = runModeConfiguration.getOptional[Int]("tai.cache.updateIncome.expiryInSeconds").getOrElse(172800)
```

Comment:
i think it would be more meaningful if u wrote it like 2*24*60*60 or something like this